nix/runtime.nix

```diff
@@ -30,6 +39,7 @@ rec {
       password = "ctl";
       db = "ctl-${network.name}";
     };
+    # These values will generate the `config.toml` required by ogmios-datum-cache
```

Comment:
Not true, we use command line arguments, not config file.

Reply:
is it ok to just remove this comment?

Reply:
Yep